### ISSUE TITLE
hydroponics?

### ISSUE THREAD

**Issue by seishedaye** [Alziibun/Perdition#36]
Maybe we can add some kind of craftable/portable farming station that uses electrical and farming so that you can essentially farm indoors? I'm not sure how easy this would be, but I think the idea would be interesting

**Comment by HaleChristopher** [comment]
Having through about this it might be a way to decrease the speed at which plants grow at the expense of water and electricity.  

Thinking of a placeable device with the following recipe:

Hydroponics Planter
Req: Farming 5
Req: Electrical 5
1 x Bucket
2 x Metal Bar
1 x Sheet
1 x Tarp
3 x Lightbulb (Am tempted to suggest 2 x Red and 1 x Blue but the latter might be an absolute pain to find) 
2 x Copper Wire
5 x Electrical Components
1 x NPK Fertiliser

(If a sound needs to be applied to it the soft hum of a forced air fan and trickle of water from a small pump would do the trick, though I might be over thinking that)

The Device would look like a bucket with a sheet over it when places, and it would be lit up internally when powered. 

Provided it was supplied with power and had water it would act as a furrow for planting and trigger the growth step in 60-75% of the time of normal planting (could be a fixed or random figure between those kind of values).

**Comment by Alziibun** [comment]
> 3 x Lightbulb (Am tempted to suggest 2 x Red and 1 x Blue but the latter might be an absolute pain to find)

Yes, let's not specify light bulb colors as this could end up being more irritating than engaging.

**Comment by HaleChristopher** [comment]
Fair enough, just my picky ass trying to get the best distribution of light for maximum growth. :-)

Does it seem like a feasible thing to make a furrow work like that?  Feels like asking a lot...

**Comment by Alziibun** [comment]
> Does it seem like a feasible thing to make a furrow work like that?  Feels like asking a lot...

A furrow act like a hydroponics tray or place a furrow inside?

**Comment by HaleChristopher** [comment]
> 

Having the Hydroponics Planter work like a furrow, but when there is no power/water the plant quickly withers
.

**Comment by Alziibun** [comment]
> > 
> 
> Having the Hydroponics Planter work like a furrow, but when there is no power/water the plant quickly withers
> .  

Yes, this will be how it's implemented
